.tx/config

```diff
@@ -0,0 +1,27 @@
+[main]
+host = https://www.transifex.com
+
+[beavy.server]
+file_filter = translations/<lang>/beavy_server.json
+source_file = var/server-messages/beavy-server.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.web]
+file_filter = translations/<lang>/beavy_web.json
+source_file = var/react-intl-messages/beavy-core.json
+source_lang = en
+type = KEYVALUEJSON
+
+[beavy.hacker-news-app]
```

Comment:
@ligthyear just following up on this: I wrote to one of the Transifex people, and she said:

> As you already know, in order to specify a resource in Transifex, you need to define both  project_slug and resource_slug. So, only this statement [project_slug.resource_slug] is acceptable and recognisable 

> I would suggest you host the different apps in different projects under your organization, as you correctly pointed out. 

Setting up distinct projects seems plausible, but it also seems as thought it could become a bit complex. Thoughts?